compiler-plugin/src/main/scala/scalapb/compiler/GrpcServicePrinter.scala

```diff
@@ -179,12 +180,18 @@ final class GrpcServicePrinter(service: ServiceDescriptor, implicits: Descriptor
 
     val grpcMethodDescriptor = "_root_.io.grpc.MethodDescriptor"
 
+    val idempotencyLevel = method.getOptions.getIdempotencyLevel
+    val safe             = idempotencyLevel == IdempotencyLevel.NO_SIDE_EFFECTS
+    val idempotent       = idempotencyLevel == IdempotencyLevel.IDEMPOTENT
```

Comment:
No side effects implies idempotent. Should there be a logical or here? https://protobuf.dev/reference/java/api-docs/com/google/protobuf/DescriptorProtos.MethodOptions.IdempotencyLevel.html#NO_SIDE_EFFECTS_VALUE

Reply:
Thanks, fixed